docs/user-interface/handlers/customize.md

```diff
@@ -1,122 +1,316 @@
+Handlers can be accessed through a control-specific interface provided by .NET MAUI, such as `IButton` for a `Button`. This avoids the cross-platform control having to reference its handler, and the handler having to reference the cross-platform control. The mapping of the cross-platform control API to the native control API is provided by a *mapper*.
+
+All handler-based .NET MAUI controls can be customized by modifying the mapper for a control's handler. The syntax for modifying a mapper is as follows:
+
+```csharp
+HandlerType.MapperName[string] = (handler, view) =>
+{
+  // Manipulate handler.NativeView to customize the handler
+};
+```
+
+All handler types are located in the `Microsoft.Maui.Handlers` namespace, and each handler provides a mapper that maps each cross-platform control property to a method that applies the property value to the native control. The `handler` argument to the mapper provides an instance of the handler being customized, and the `view` argument provides an instance of the cross-platform control that the handler implements. For a list of the type names that implement handler-based .NET MAUI controls, see [Handler-based controls](#handler-based-controls).
 
-Handlers expose the native control that implements the .NET MAUI cross-platform control via the `NativeView` property. This property can be accessed to set platform properties, invoke platform methods, and subscribe to platform events.
+Each handler class exposes the native control that implements the cross-platform control via its `NativeView` property. This property can be accessed to set native control properties, invoke native control methods, and subscribe to native control events. In addition, the cross-platform control implemented by the handler is exposed via its `VirtualView` property.
 
-Handlers are global and therefore customization can occur anywhere in your .NET MAUI app. For example, customizing a handler for a control in your `App` class will result in all controls of that type being customized in the app. Similarly, customizing a handler for a control in the first page of your app will also result in all controls of that type in the app being customized.
+Handlers can be customized to augment the appearance and behavior of a .NET MAUI control beyond the customization that's possible through the control's API. Handlers are global and therefore customization can occur anywhere in your .NET MAUI app. For example, customizing a handler for a control in your `App` class will result in all controls of that type being customized in the app. Similarly, customizing a handler for a control in the first page of your app will also result in all controls of that type in the app being customized.
```

Comment:
Maybe be more direct instead of using examples to demonstrate the concept? Maybe you want to tweak this further or ignore it? Just edit my comment here. Is **view** the right term? I'm unsure...
```suggestion
Handlers can be customized to augment the appearance and behavior of a .NET MAUI control beyond the customization that's possible through the control's API. Handler customizations are global and aren't scoped to a specific UI view. Handler customization is allowed to happen anywhere in your app. Once a handler is customized, it affects all native controls of that type, everywhere in your app.
```